app/models/Broadcast.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/**
+ * Imports.
+ */
+const mongoose = require('mongoose');
+
+/**
+ * Schema.
+ */
+const broadcastSchema = new mongoose.Schema({
+
+  _id: { type: Number, index: true },
+  type: { type: String, enum: ['signup_prompt'] },
+  campaign: Number,
+  messages: {
+    signup_prompt_declined: String,
```

Comment:
I'm using the nested `messages` convention that was introduced in #763 -- but I'm not entirely convinced this is the way to go right now since we use our Mongo documents as a CMS. Nesting it as an object makes it a little tricker for our admins to manually edit.